apps/asap-server/test/utils/migrations.test.ts

```diff
@@ -0,0 +1,84 @@
+import { RestUser, Results, Squidex } from '@asap-hub/squidex';
+import { applyToAllItemsInCollection } from '../../src/utils/migrations';
+
+const mockFetch: jest.MockedFunction<Squidex<RestUser>['fetch']> = jest.fn();
+const mockConstructor = jest.fn();
+
+jest.mock('@asap-hub/squidex', () => ({
+  Squidex: class Squidex {
+    constructor(collection: string) {
+      mockConstructor(collection);
+    }
+    fetch = mockFetch;
+  },
+}));
+
+const restUserMock = {
+  id: 'userId',
+  onboarded: true,
+  displayName: 'Cristiano Ronaldo',
+  created: '',
+  lastModified: '',
+  data: {},
+} as any as RestUser;
```

Comment:
How come this had to be casted?

Reply:
because of missing `data`, it just felt unnecessary to add it all in line

yeah fixtures are a pain, i would like to have fixture generators for most of the return types and then used across other generators when nesting is required but oh well, i dont think this is the right time to refactor

Reply:
At least we removed an additional fixture. When should this tech debt be addressed? I think doing it in parts during other tickets is the best way forward. Tech debt tickets don't provide any user value (usually)